src/component/Categories.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      Categories
+    </div>
```

Comment:
- [ ] Good job working on the categories page, but please kindly ensure it has a button with the text `Check Status`